File: vocabular-learning-app-ui/src/app/models/word.ts
export class Word {

    public wordId : number | undefined;
    public word: string = "";
    public phonetics: string = "";
    public synonyms: string = "";
    public antonyms: string = "";
    public notes: string = "";
    public wordMeanings:string[] = [];
    public createdTimeStamp: Date | undefined;
    public updatedTimeStamp: Date | undefined;

}

File: vocabulary.learning.app/src/main/java/com/vocabulary/learning/app/model/Word.java
package com.vocabulary.learning.app.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.time.LocalDateTime;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Setter
@Getter
@ToString
public class Word {
    private Integer wordId;
    private String word;
    private String phonetics;
    private String synonyms;
    private String antonyms;
    private String notes;
    private String meaning;
    private LocalDateTime createdTimeStamp;
    private LocalDateTime updatedTimeStamp;

}
